.circleci/config.yml

```diff
@@ -32,10 +32,18 @@ jobs:
           key: gradle-{{ checksum "build.gradle.kts" }}
       - store_test_results:
           path: build/test-results
+      - store_test_results:
```

Comment:
In circleci, we need to store a 2nd set of test results, because we now have 2 build paths (one for the. main module/app, one for the domain timeline source); and each generates it's own set of build files including test reports.